### DISCUSSION TITLE
Synology deploy errors acme.sh

### DISCUSSION THREAD

**Discussion by Glasairmell** [acmesh-official/acme.sh#6162] [Q&A]
Howdy
When I deploy the cert  ./acme.sh -d "*.xxxxx.com" --deploy --deploy-hook synology_dsm --home $PWD

I am getting these errors.

[Thu Dec 12 06:52:05 PM PST 2024] The domain '*.xxx.com' seems to already have an ECC cert, let's use it.
[Thu Dec 12 06:52:06 PM PST 2024] Logging into localhost:5000...
[Thu Dec 12 06:52:06 PM PST 2024] Getting certificates in Synology DSM...
[Thu Dec 12 06:52:07 PM PST 2024] Unable to find certificate: Let’s Encrypt and  is not set.
[Thu Dec 12 06:52:07 PM PST 2024] Error deploying for domain: *.xxxxxx.com
[Thu Dec 12 06:52:07 PM PST 2024] Error encountered while deploying.

Any thoughts please on how to fix this?

Cheers

**Comment by Glasairmell** [comment]
This was a Synology permission problem which I fixed.  Deployed perfectly after that.  Wonderful script and much appreciated.

**Reply by hahfwang** [reply]
Could you please let me know how you solved the issue?
